tests/pages/externalpages/pocket/base.py

```diff
@@ -0,0 +1,101 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from pypom import Page, Region
+from selenium.webdriver.common.by import By
+
+
+class ScrollElementIntoView:
+    def scroll_element_into_view(self, strategy, locator):
+        # scroll elements so they are not beneath the navigation
+        offset = {"x": 0, "y": -100}
+        return super(ScrollElementIntoView, self).scroll_element_into_view(strategy, locator, **offset)
+
+
+class BaseRegion(ScrollElementIntoView, Region):
+    pass
+
+
+class BasePage(ScrollElementIntoView, Page):
+    def __init__(self, selenium, base_url, locale="en-US", **url_kwargs):
+        super(BasePage, self).__init__(selenium, base_url, locale=locale, **url_kwargs)
+
+    def wait_for_page_to_load(self):
+        browser_name = self.selenium.capabilities.get("browserName").lower()
+        browser_version = self.selenium.capabilities.get("browserVersion") or self.selenium.capabilities.get("version")
+        is_ie9 = browser_name == "internet explorer" and browser_version == "9"
```

Comment:
We can also probably remove the IE9 checks here? The Pocket pages don't work in old IE browsers (from what I can tell when testing in prod). I'm assuming that's OK, so if so - we don't need to run tests against them in IE9?